samcli/hook_packages/terraform/hooks/prepare/hook.py

```diff
@@ -141,6 +141,11 @@ def prepare(params: dict) -> dict:
         if cfn_dict.get("Resources"):
             _update_resources_paths(cfn_dict.get("Resources"), terraform_application_dir)  # type: ignore
 
+        # Add hook metadata
+        cfn_dict["Metadata"]["AWS::SAM::Hook"] = {
```

Comment:
could you please define constant variables for these values

Reply:
AWS::SAM::Hook makes it appear as if its a CFN deployable resource, is this intentional?

Reply:
we just want to come with a name to be similar to the one we used for SAR Application metatdata https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-template-publishing-applications-metadata-properties.html .. but we can change it for sure

Reply:
>could you please define constant variables for these values

good call, done